### PR TITLE
fix js parsing int64

### PR DESCRIPTION
**Pull request by Akiq2016**
> All Numbers in Javascript are 64 bit "double" precision IEE754 floating point.
The largest positive whole number that can therefore be accurately represented is 2^53 - 1. The remaining bits are reserved for the exponent.

### CODE REVIEW

**Comment by tmc** [comment]
I presume this would change the generated examples. Can you do so in another commit to this branch?

Please see the makefile for the relevant target.

**Comment by tmc** [comment]
@Akiq2016 I noticed this was your first go contribution and I wanted to make it easier so I went ahead and pushed the example update. Need some sleep but I'll finish reviewing this in the morning.

**Comment by tmc** [comment]
I think some folks would want to override this behavior and keep it number (and risk bad representation of very large numbers) -- do you think you could take a crack at adding an optional argument that implements the old behavior? Organizing the two groups of numbers in the right order in the switch would allow use of fallbacks and minimize branching/conditionals.

**Comment by Akiq2016** [comment]
thanks and let me try~

File: protoc-gen-tstypes/main.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ var (
 	flagOriginalNames         = flag.Bool("original_names", false, "if true, use original proto file field names, otherwise convert to lowerCamelCase")
 	flagOutputFilenamePattern = flag.String("outpattern", "{{.Dir}}/{{.Descriptor.GetPackage | default \"none\"}}.{{.BaseName}}.d.ts", "output filename pattern")
 	flagDumpDescriptor        = flag.Bool("dump_request_descriptor", false, "if true, dump request descriptor")
+	flagInt64AsString         = flag.Bool("int64_string", false, "if true, generate string int64")
```
Comment (tmc) [inline protoc-gen-tstypes/main.go]: can this be worded "if true, use string representation for 64 bit numbers"

**Comment by tmc** [comment]
@Akiq2016 this is great! I think the only thing left is updating the docs at https://github.com/tmc/grpcutil/blob/master/protoc-gen-tstypes/doc.go#L24 (which get rendered at https://godoc.org/github.com/tmc/grpcutil/protoc-gen-tstypes#hdr-Options )

**Comment by Akiq2016** [comment]
thank you for your patient! ◝(⑅•ᴗ•⑅)◜..°♡

**Comment by tmc** [comment]
@Akiq2016 this is great! Thanks so much! 💯

**Pull request merged**